modules/op-resolver-bundle.adoc

```diff
@@ -121,3 +121,28 @@ spec:
   - name: sample-task-parameter
     value: test
 ----
+
+The following example task includes a step that references a `StepAction` definition from a Tekton bundle:
+
+[source,yaml]
```

Comment:
It seems a bit non-standard to have big chunks of content detached from the list after the `.Procedure`. There are a handful of examples in the PR of other procedures like this, too, but I'll only mention here.

I think I would have expected this to just be appended to the previous step with `+`s as fuller examples, or broken off into an "Examples" subsection of this procedure module.